src/pages/UserSettings.vue

```diff
@@ -1,27 +1,18 @@
 <template>
   <nav-bar />
-  <div class="container-fluid my-3 d-flex justify-content-center">
+  <div class="container-fluid my-3 d-flex justify-content-center mt-5">
     <div class="card col-12 col-sm-8 col-md-6 col-lg-7 col-xl-4 border-0">
-      <div class="card-body " style="background-color: #17202A;">
+      <div class="card-body bg-dark-subtle border-0 rounded-lg shadow-dark">
         <h4 class="card-title text-center">
           Configuración de Perfil
         </h4>
         <div class="row px-3">
           <div class="d-flex justify-content-center my-3">
-            <div class="style-avatar shadow-lg" :style="{ 'background-image': 'url(' + userObj.avatar.image + ')' }">
-              <input
-                ref="fileImageProfile"
-                type="file"
-                hidden
-                @change="uploadImage"
-              >
-              <div
-                class="content-save d-flex justify-content-center align-items-center shadow-lg rounded-circle"
-                @click="eventSelectedImage"
-              >
-                <i class="bi bi-camera-fill" />
-              </div>
-            </div>
+            <img
```

Comment:
didnt we say that were were going to create a bunch of personas icons and they should be able to chose a color around their icons.